src/app/shared/collapsible-sidebar/collapsible-sidebar.component.scss

```diff
@@ -9,6 +9,10 @@
   overflow: auto;
 }
 
+.toggle {
+  opacity: 0.6;
```

Comment:
I suggest to have opacity 1.0 when hover. WDYT?

Reply:
Ok